packages/eas-cli/src/commands/build/index.ts

```diff
@@ -1,9 +1,132 @@
-import { Command } from '@oclif/command';
+import { Command, flags } from '@oclif/command';
+import fs from 'fs-extra';
+import path from 'path';
+import { v4 as uuidv4 } from 'uuid';
 
-export default class BuildIndex extends Command {
-  static description = 'build an app binary for your project';
+import { configureAsync } from '../../build/configure';
+import { createCommandContextAsync } from '../../build/context';
+import { buildAsync } from '../../build/create';
+import { AnalyticsEvent, RequestedPlatform } from '../../build/types';
+import Analytics from '../../build/utils/analytics';
+import { isGitStatusCleanAsync } from '../../build/utils/repository';
+import {
+  isEasEnabledForProjectAsync,
+  warnEasUnavailable,
+} from '../../project/isEasEnabledForProject';
+import { findProjectRootAsync, getProjectIdAsync } from '../../project/projectUtils';
+import { confirmAsync, promptAsync } from '../../prompts';
+import { ensureLoggedInAsync } from '../../user/actions';
 
-  async run() {
-    this._help();
+export default class BuildCreate extends Command {
+  static description = 'Start a build';
+
```

Comment:
Add `build:create` alias?

Reply:
I agree with https://github.com/expo/eas-cli/pull/105#issuecomment-738186983
help output is confusing with alias